docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -448,6 +448,15 @@ The following command operators are supported:
 
   Renders a numeric value in typed JSON logs.
 
+%UPSTREAM_CONNECTION_POOL_CALLBACK_DURATION%
+  HTTP/TCP
+    Total duration in milliseconds from when the upstream request was created to when the connection pool callbacks.
```

Comment:
nit: `UPSTREAM_CONNECTION_POOL_READY_DURATION`

`Total duration in milliseconds from when the upstream request was created to when the connection pool is ready.`

End users may don't care the internal `callback`, tell him what is `ready` is more straightforward.